File: tsodd-service/src/main/java/ru/thetenzou/tsoddservice/controller/Health.java
package ru.thetenzou.tsoddservice.controller;

import java.util.Collections;
import java.util.Map;

import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
@RequestMapping("/api/tsodd")
public class Health {

    @GetMapping("/health")
    public Map getHealth() {
        return Collections.singletonMap("status", "live");
    }
    
}

File: tsodd-service/src/main/kotlin/ru/thetenzou/tsoddservice/common/controller/Health.kt
package ru.thetenzou.tsoddservice.common.controller

import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RequestMapping
import org.springframework.web.bind.annotation.RestController
import java.util.*


@RestController
@RequestMapping("/api/tsodd")
class Health {

    @GetMapping("/health")
    fun health() = Collections.singletonMap("status", "live")

}
